packages/pg-cursor/package.json

```diff
@@ -15,8 +15,10 @@
   },
   "author": "Brian M. Carlson",
   "license": "MIT",
-  "devDependencies": {
-    "mocha": "^7.1.2",
+  "dependencies": {
```

Comment:
To be a peer dependency, it should be in `"peerDependencies"`.

Reply:
done